style.css

```diff
@@ -171,9 +215,48 @@ h1 {
         width: 15vh;
         transform: translateX(-50%) translateY(-50%);
     }
+
+    /* WHY-SECTION TABLET */
+    .why {
+        flex-direction: row;
+        flex-wrap: wrap;
+        padding: 0px 10px;
+        background-image: url('img/desktop/honey.png');
+        background-repeat: no-repeat;
+        background-position-x: 33em;
+        background-position-y: 60em;
+        background-size: 700px;
+    }
+
+    .why-box {
+        width: 45%;
+        height: 28vh;
+        margin: 0px;
+        padding: 0px 10px;
+    }
+
+    .why-h2 {
+        font-size: 75px;
+        width: 100%;
+    }
+
+    .why-img {
+        width: 60%;
+    }
+
+    .why-h3 {
+        font-size: 48px;
```

Comment:
Font sizes are best set in `rem` or `em` for accessibility :)

Reply:
If you decide to use Sass (which I recommend, as you can get the output file compressed → smaller file size → better page loading speed & more sustainable energy resource consumption), you can create a function that makes the conversion easier :) Like this:

```
@function rem($px) { @return ($px / 16px) * 1rem; }
// Use:
font-size: rem(48px);
```